spring-batch-core-tests/src/test/java/test/jdbc/datasource/DerbyDataSourceFactoryBean.java

```diff
@@ -38,7 +38,7 @@ protected DataSource createInstance() throws Exception {
 		System.setProperty("derby.storage.pageCacheSize", "100");
 
 		final EmbeddedDataSource ds = new EmbeddedDataSource();
-		ds.setDatabaseName("derbydb");
+		ds.setDatabaseName("build/derbydb");
```

Comment:
Even though this is a setter for the database "name", without this change the `derbydb` folder will be created under the root directory of the module and not under `build` directory.